ospgrillage/mesh.py

```diff
@@ -767,7 +773,7 @@ def _orthogonal_meshing(self):
                     self.ortho_previous_node_column = self.assigned_node_tag
                     self.assigned_node_tag = []
 
-                print("Edge mesh @ start span completed")
+                # print("Edge mesh @ start span completed")
```

Comment:
delete line? Or if debugging output is required then add a `verbose` option for the user to switch on/off such outputs?